src/Equinox.Cosmos/Cosmos.fs

```diff
@@ -939,16 +917,18 @@ type private Folder<'event, 'state>
         ?readCache) =
     let inspectUnfolds = match mapUnfolds with Choice1Of3 () -> false | _ -> true
     interface ICategory<'event, 'state, Container*string> with
-        member __.Load containerStream (log : ILogger): Async<StreamToken * 'state> =
-            let batched = category.Load inspectUnfolds containerStream fold initial isOrigin log
-            let cached tokenAndState = category.LoadFromToken tokenAndState fold isOrigin log
-            match readCache with
-            | None -> batched
-            | Some (cache : Caching.Cache, prefix : string) ->
-                match cache.TryGet(prefix + snd containerStream) with
-                | None -> batched
-                | Some tokenAndState -> cached tokenAndState
-        member __.TrySync (log : ILogger) (streamToken,state) (events : 'event list)
+        member __.Load containerStream (log : ILogger): Async<StreamToken * 'state>  = async {
+                let! batched = category.Load inspectUnfolds containerStream fold initial isOrigin log
+                let cached tokenAndState = category.LoadFromToken tokenAndState fold isOrigin log
+                match readCache with
+                | None -> return batched
+                | Some (cache : ICache, prefix : string) ->
+                    let! cacheItem = cache.TryGet(prefix + snd containerStream)
+                    match cacheItem with
+                    | None -> return batched
+                    | Some tokenAndState -> return! cached tokenAndState
+            }
```

Comment:
In this file, I put the brace on prev line for conservation of vertical space (you don't have to, but don't be surprised if I move it up later!)